### PR TITLE
Update MVC migrating config doc to RC1

### PR DESCRIPTION
**Pull request by scottaddie**
This PR addresses issue https://github.com/aspnet/Docs/issues/847. In addition to the RC1 upgrade, the corresponding sample solution has been added.

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
Done

File: mvc/migration/migratingconfig.rst [hunk 1]
```diff
@@ -1,81 +1,55 @@
...
 
-ASP.NET 5 and ASP.NET MVC 6 no longer use the Global.asax and Web.config files that previous versions of ASP.NET utilized. In earlier versions of ASP.NET, application startup logic was placed in an Application_StartUp() method within Global.asax. Later, in ASP.NET MVC 5, a Startup.cs file was included in the root of the project, and was called using an OwinStartupAttribute when the application started. ASP.NET 5 (and ASP.NET MVC 6) have adopted this approach completely, placing all startup logic in the Startup.cs file.
+The *web.config* file has also been replaced in ASP.NET 5. Configuration itself can now be configured, as part of the application startup procedure described in *Startup.cs*. Configuration can still utilize XML files, if desired, but typically ASP.NET 5 projects will place configuration values in a JSON-formatted file, such as *appsettings.json*. ASP.NET 5's configuration system can also easily access environment variables, which can provide a more secure and robust location for environment-specific values. This is especially true for secrets like connection strings and API keys that should not be checked into source control.
```
Comment (Rick-Anderson) [inline mvc/migration/migratingconfig.rst]: delete ",if desired"

File: mvc/migration/migratingconfig.rst [hunk 2]
```diff
@@ -1,81 +1,55 @@
...
+	using Microsoft.Extensions.Configuration;
 
-	public Startup(IHostingEnvironment env, IApplicationEnvironment appEnv)
-	{
-		var configurationBuilder = new ConfigurationBuilder()
-			.SetBasePath(appEnv.ApplicationBasePath)
-			.AddJsonFile("config.json");
-		Configuration = configurationBuilder.Build();
-	}	
+Finally, add an *appsettings.json* file to the root of the project using the appropriate item template:
```
Comment (Rick-Anderson) [inline mvc/migration/migratingconfig.rst]: remove Finally,

File: mvc/migration/migratingconfig.rst [hunk 3]
```diff
@@ -1,81 +1,55 @@
...
-	{
-		"Data": {
-			"DefaultConnection": { 
-				"ConnectionString": "Server=(localdb)\\MSSQLLocalDB;Database=_CHANGE_ME;Trusted_Connection=True;"
-			}
-		}
-	}
-
-Change the name of the Database from _CHANGE_ME. In the case of this migration, we are going to point to a new database, which we'll name NewMvc6Project to match our migrated project name.
+In the highlighted line depicted above, change the name of the database from **_CHANGE_ME**. In the case of this migration, we are going to point to a new database, which we'll name *NewMvc6Project* to match our migrated project name.
```
Comment (Rick-Anderson) [inline mvc/migration/migratingconfig.rst]: For this tutorial, we won't migrate the previous database, we'll create a new one. Name it *NewMvc6Project* to match our migrated project name.

**Comment by Rick-Anderson** [comment]
Looks good. A couple minor tweaks. :watch:

File: mvc/migration/migratingconfig.rst (hunk 3 above)
Comment (scottaddie) [inline mvc/migration/migratingconfig.rst]: @Rick-Anderson How does the following sound?

In the highlighted line depicted above, change the name of the database from **_CHANGE_ME**. We are going to point to a new database, which will be named *NewMvc6Project* to match our migrated project name.

Comment (Rick-Anderson) [inline mvc/migration/migratingconfig.rst]: perfect

**Comment by scottaddie** [comment]
@Rick-Anderson All checked in. Please take another look.

**Comment by Rick-Anderson** [comment]
:shipit: @shirhatti please merge

**Comment by scottaddie** [comment]
@Rick-Anderson Should this PR be left open until it's merged? Not sure if that was done for a reason or not.

**Comment by Rick-Anderson** [comment]
Not sure how I closed that. :shipit: @shirhatti please merge

**Comment by shirhatti** [comment]
Thanks! Merged in e968e9c2c7e928c4b5b0df8f1c63622b3ccd4dba

**Pull request closed without merging**